build.sbt

```diff
@@ -118,11 +169,17 @@ lazy val akkaHttpServer: Project = (project in file("server/akka-http-server"))
   )
   .dependsOn(core, serverTests % "test")
 
+
 lazy val http4sServer: Project = (project in file("server/http4s-server"))
   .settings(commonSettings: _*)
   .settings(
     name := "http4s-server",
-    libraryDependencies ++= Seq("org.http4s" %% "http4s-blaze-server" % http4sVersion)
+    libraryDependencies ++= Seq(
+      "org.http4s" %% "http4s-blaze-server" % http4sVersion,
+      "org.http4s" %% "http4s-dsl"          % http4sVersion
+    ),
+    scalacOptions ++= Seq("-Ypartial-unification") ++ scalacLintOptions,
+    addCompilerPlugin("org.spire-math" %% "kind-projector" % "0.9.8"),
```

Comment:
this should be provided by sbt-softwaremill, or maybe it doesn't work for you?

Reply:
Kind projector? Didn't check, copied this from my separate repo :)

Reply:
ok, I think all of these (options & kind-projector) are provied by the plugin :)